### PR TITLE
[Ready for review] Create new readme

### PR DESCRIPTION
**Pull request by simonkirbysimonkirby**
This PR:
* Adds a semi useful README.md to the project with instructions for running the site locally.
* Adds a .gitignore file from Ruby and Jekyll templates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,25 @@
...
 
-To create a new website for a new workshop,  please use GitHub's "template" function to copy this `UoE-workshop-template` repository.
-**Please _do not fork this repository directly on GitHub._**
+This repository is based off of the University of Exeter `UoE-workshop-template` repository. This template uses Jekyll and GitHub pages to create a workshop guide.
 
-## Structure and required components
+## Ruby & local testing
 
-Each workshop has a landing or home page which provides the details of the workshop to individuals who might be interested in attending it. This home page will also direct participants who have signed up to set-up instructions and the course notes. It will also enable individuals who can't attend the workshop to all the resources and enable them to run through the material in their own time. 
+Jekyll is a ruby Gem. A simple Gemfile has been added. To test the site locally, clone the repository, and run the following (on macOS). Ensure `ruby` and `bundle` have been installed, and then run the following from the project root:
```
Comment (stephenpcook) [inline README.md:13]: bundler, not bundle.

Comment (stephenpcook) [inline README.md:13]: Works fine on Windows, although I didn't use the `bundle config` line.

File: README.md [hunk 2]
```diff
@@ -1,28 +1,25 @@
...
 
-Workshops are organised into 2-3 hour sessions of learning activity, where a workshop consists of 1-4 sessions. Beginners courses should have a minimum of 2 sessions.  Each session will be split into episodes, where episodes should last between 20-90 mins. Workshops and sessions will follow a pre-specified format. We recommend that there is an opportunity for a break approximately every 60-90 mins. 
+`bundle config set --local path 'vendor/bundle'`
```
Comment (stephenpcook) [inline README.md:15]: Add these three commands in a single fenced block:
```sh
bundle config set --local path 'vendor/bundle'
bundle install
bundle exec jekyll serve
```
or at least the last two (if the first is a MacOS thing)

**Review by stephenpcook — CHANGES_REQUESTED** [review]
Minor changes of typos and some formatting suggestions.

**Comment by simonkirbysimonkirby** [comment]
@stephenpcook Thanks for the review, going to merge this one.

**Pull request merged**